tests/dummy/app/components/main-navigation.js

```diff
@@ -8,6 +8,7 @@ export default Component.extend({
 
   actions: {
     login() {
+      // eslint-disable-next-line ember/closure-actions
       this.sendAction('onLogin');
```

Comment:
Can we refactor this? We shouldn't be using old patterns in the dummy app really…